### PR TITLE
Created Contributor Guide and Bug Reports Proposal 

### PR DESCRIPTION
**Pull request by Sophyia7**
I Created a Contributor Guide and Bug Reports Proposal. This is t help users report any issue easily and be part of the project.

### CODE REVIEW

**Review by shivay-at-pieces — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discord Community Support
+    url: https://discord.gg/getpieces
+    about: You can ask and answer questions here.
+  - name: Documentation
+    url: https://docs.pieces.app/
+    about: Pieces documentation.
```
Comment (jordan-pieces) [inline .github/ISSUE_TEMPLATE/config.yml:8]: @Sophyia7 we should add a little more to this description here

**Review by jordan-pieces — CHANGES_REQUESTED** [review]
looks great lets just get some more added to the description on the link to the docs site and then we should be good to go ! 💯

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Sophyia7) [reply .github/ISSUE_TEMPLATE/config.yml]: @jordan-pieces I adjusted it, and I also added our YT page.

**Comment by jordan-pieces** [comment]
@Sophyia7 it looks like we are having a similar issue here on this PR as we did on the Python project - due to the dist size and updates to the package itself, you need to update your fork from main and then update your branch, or you can just open a new PR with your copied configuration files. 

Just wanted to add this for the future, im going to take care of this today to get this merged 👍 not sure why we keep having these issues and going forward we may benefit from including the /dist directory in the .gitignore. Some would have to just run the generation locally but could be a better option.

- [ ] research: Should i add my dist to my gitignore file?

**Pull request closed without merging**